Algebraic Topology/algebraic-topology.tex

```diff
@@ -1051,6 +1050,74 @@ \section{CW complexes and the fundamental group}
+    \item \includegraphics[scale=0.2]{images/not-delta-complex3}
+  \end{itemize}
+  $\Delta$-complexes allow you to glue vertices together, but simplicial complexes do not. ((John Harnois))
+\end{rmk}
+
+An $n$-chain (non-rigorous idea) is a union (with multiplicity, and negative multiplicity) of $n$-simplices.  Therefore, ann $n$-chain is a subspace of dimension $n$.  Also, negative multiplicities represent the opposite orientation of their positive counterparts.
+
+An orientation of an $n$-simplex is specified by an order on the vertices.  Also, an even permutation (on these vertices) is orientation preserving, while an odd permutation is orientation reversing.  By convention in Hatcher (and this course), faces are oriented according to the induced order from their vertices.
+
+((Hatcher's definition, which makes sense to me:))
+\begin{defn}
+  Let \de{$\Delta_n(X)$} be the free abelian group with basis the open $n$-simplices $e_\alpha^n$ of $X$.  Then an \de{$n$-chain} is an element of $\Delta_n(X)$, and is written $\sum_\alpha n_\alpha e_\alpha^n$ with coefficients $n_\alpha \in \Z$.  Equivalently, we could write $\sum_\alpha n_\alpha \sigma_\alpha$ where $\sigma_\alpha \from \Delta^n \to X$ is the characteristic map ((glueing map??)).
+\end{defn}
+((Websters definition:))
+\begin{defn}
+  An \de{$n$-chain} $\Delta_n(X)$ of a $\Delta$-complex $X = \dunion \Delta_{\alpha}^n /$\text{every face $\cong$ a lower $d$-simplex}  (Yes, mod out by that) is a  formal sum w/ coefficients in $\Z$ of $n$-simplices.
+\end{defn}
+
+\begin{defn}[boundary of an oriented $n$-simplex]
+  The boundary of an $n$-simplex $[0, 1,,, n]$ is defined by $$\d([0, 1,,, n]) := \sum_{i=0}^n (-1)^i [0, 1,,, \hat{i},,, n]$$
+\end{defn}
+\begin{defn}[boundary of an $n$-chain]
+  The boundary map $\d \from \Delta_n(X) \to \Delta_{n-1}(X)$ is linear, and so $$\d(\sum_\alpha n_\alpha e_\alpha^n) := \sum_\alpha n_\alpha \d(e_\alpha^n)$$
+\end{defn}
+((i defined this differently than webster.  i'm guessing this is the same.))
+
+Houston we have a problem.  The induced orientation on the boundary doesn't neccessarily agree with the original orientation on the those lower dimensional simplices!  (See above).  So what to do?  I think that's where the $(-1)^i$ comes into play.
+
+\begin{example}[Hatcher, p.105]
+  \quad
+
+  \includegraphics[scale=0.5]{images/boundary-of-oriented-simplex}
+\end{example}
+
+\begin{example}
+  % pretty picture
+  $$\d(X) = - a - b + c$$
+  $$\d(Y) =$$
+
+  $\sigma$ is the glueing map!  not a permutation! $\sigma \from \Delta^n \to X$.
+  where $[0, 1, 2]$ denotes the simplex.
+\end{example}
+
+Claim: $\d^2 = 0$.
+
+\begin{defn}
+  $$H_n^\Delta(X) := \ker(\d \from \Delta_n \to \Delta_{n-1})/\im(\d \from \Delta_{n+1} \to \Delta_{n})$$
+\end{defn}
+
+\begin{example}
+  $H_2^\Delta(K^2) = 0$
+  $H_1^\Delta(K^2) \isom \Z \oplus \Z/2\Z$
+  $H_0^\Delta(K^2) = \Z$
+
+  where
+  $\Z^2 \to(overset [-1 -1; -1 1; 1 1]) \Z^3 \to(overset 0) \Z$
```

Comment:
I am not quite sure what the syntax is here, but it does not look good in the pdf.